tests/common/mod.rs

```diff
@@ -118,6 +119,15 @@ pub fn keepldr_exec_crate<'a>(
         .unwrap_or_else(|e| panic!("failed to run `{}`: {:#?}", bin, e))
         .unwrap_or_else(|| panic!("process `{}` timed out", bin));
 
+    if let Some(input_thread) = input_thread {
+        if let Err(_) = input_thread.join() {
+            unsafe {
+                eprintln!("{}", String::from_utf8_unchecked(output.stderr));
```

Comment:
I'm curious about the use of `from_utf8_unchecked` here as opposed to just `from_utf8`.

Reply:
oops.. debug leftover. Thanks!

Should be:
```rust
let _unused = stderr().write_all(&output.stderr);
```